website/docs/learning-and-resources/tutorials/review-moderator-dashboard/creating-interactive-views-using-lists-and-charts.md

```diff
@@ -165,6 +165,7 @@ There are almost 100+ variants of Fusion Chart Configuration; learn more from th
 Appsmith uses the Chart Series property head to supply the data and details related to identifying the data points.
 
 * `Series Title` - the name of the series.
+* `Series Color` - a color picker input to select the color of the series (this property is not available for pie chart yet).
```

Comment:
Let's not add the property here, as it is a separate tutorial and will need additional work to be changed.